Jellyfin.Server.Implementations/Devices/DeviceManager.cs

```diff
@@ -167,38 +174,38 @@ public async Task<QueryResult<DeviceInfo>> GetDeviceInfos(DeviceQuery query)
         }
 
         /// <inheritdoc />
-        public async Task<QueryResult<DeviceInfo>> GetDevicesForUser(Guid? userId)
+        public QueryResult<DeviceInfo> GetDevicesForUser(Guid? userId)
         {
-            var dbContext = await _dbProvider.CreateDbContextAsync().ConfigureAwait(false);
-            await using (dbContext.ConfigureAwait(false))
-            {
-                var sessions = dbContext.Devices
-                    .Include(d => d.User)
-                    .OrderByDescending(d => d.DateLastActivity)
-                    .ThenBy(d => d.DeviceId)
-                    .SelectMany(d => dbContext.DeviceOptions.Where(o => o.DeviceId == d.DeviceId).DefaultIfEmpty(), (d, o) => new { Device = d, Options = o })
-                    .AsAsyncEnumerable();
+            var devices = _devices.Values
+                .OrderByDescending(d => d.DateLastActivity)
+                .ThenBy(d => d.DeviceId)
+                .AsEnumerable();
```

Comment:
Redundant, it already is an IEnumerable

Reply:
It is an `IOrderedEncumerable` without this which cannot be implicitly converted to  `IEnumerable`

Reply:
But an IOrderedEnumerable implements IEnumerable and the implementation for AsEnumerable just returns it as-is.

Reply:
Other methods like `.Where` and `.Select` does not return `IOrderedEnumerable` so we need to explicitly declare the type of `devices` as `IEnumerable<Device>`, but should work